src/app/utils/field-validation/answerValidator.factory.ts

```diff
@@ -76,6 +78,8 @@ export const constructSingleAnswerValidator = (
     return constructDropdownValidator(formField)
   } else if (isEmailField(formField)) {
     return constructEmailValidator(formField)
+  } else if (isUenValid(formField)) {
```

Comment:
this should be `isUenField` right?

Reply:
Ah good catch, yes! Let me fix that.

Reply:
actually I believe all the tests will pass once you fix this

Reply:
I've pushed out the update man.